ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,264 @@
+  isLocal?: boolean;
+  sections: ReadonlyArray<ServicesSectionState>;
+  isRefreshing: boolean;
+  onRefresh: (hideToast: boolean) => void;
+  onServiceSelect: (service: ServicePublic) => void;
+  handleOnLongPressItem: () => void;
+  isLongPressEnabled: boolean;
+  dispatchUpdateOrganizationsOfInterestMetadata?: (
+    selectedItemIds: Option<Set<string>>
+  ) => void;
+  onItemSwitchValueChanged: (service: ServicePublic, value: boolean) => void;
+  tabOffset: Animated.Value;
+}>;
+
+type Props = OwnProps &
+  ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps> &
+  LightModalContextInterface;
+
+const styles = StyleSheet.create({
+  activeTextStyle: {
+    ...makeFontStyleObject(Platform.select, "600"),
+    fontSize: Platform.OS === "android" ? 16 : undefined,
+    fontWeight: Platform.OS === "android" ? "normal" : "bold"
+  },
+  textStyle: {
+    fontSize: customVariables.fontSizeSmall
+  },
+  organizationLogo: {
+    marginBottom: 0
+  },
+  icon: {
+    paddingHorizontal: (24 - 17) / 2 // (io-right icon width) - (io-trash icon width)
+  }
+});
+
+class ServicesTab extends React.PureComponent<Props> {
+  /**
+   * For tab Locals
+   */
+  private renderOrganizationLogo = (organizationFiscalCode: string) => {
+    return (
+      <OrganizationLogo
+        logoUri={getLogoForOrganization(organizationFiscalCode)}
+        imageStyle={styles.organizationLogo}
+      />
+    );
+  };
+
+  private organizationContainsText(item: Organization, searchText: string) {
+    return isTextIncludedCaseInsensitive(item.name, searchText);
+  }
+
+  private showChooserAreasOfInterestModal = () => {
+    const {
+      selectableOrganizations,
+      hideModal,
+      selectedOrganizations
+    } = this.props;
+    this.props.showModal(
+      <ChooserListContainer<Organization>
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
+        keyExtractor={(item: Organization) => item.fiscalCode}
+        itemTitleExtractor={(item: Organization) => item.name}
+        itemIconComponent={left((fiscalCode: string) =>
+          this.renderOrganizationLogo(fiscalCode)
+        )}
+        onCancel={hideModal}
+        onSave={this.onSaveAreasOfInterest}
+        isRefreshEnabled={false}
+        matchingTextPredicate={this.organizationContainsText}
+        noSearchResultsSourceIcon={require("../../../img/services/icon-no-places.png")}
+        noSearchResultsSubtitle={I18n.t("services.areasOfInterest.searchEmpty")}
+      />
+    );
+  };
+
+  private onSaveAreasOfInterest = (
+    selectedFiscalCodes: Option<Set<string>>
+  ) => {
+    if (this.props.dispatchUpdateOrganizationsOfInterestMetadata) {
+      this.setState({
+        toastErrorMessage: I18n.t(
+          "serviceDetail.onUpdateEnabledChannelsFailure"
+        )
+      });
+      this.props.dispatchUpdateOrganizationsOfInterestMetadata(
+        selectedFiscalCodes
+      );
+      this.props.hideModal();
```

Comment:
Perhaps _**hideModal**_ should be done anyway not only if `if (this.props.dispatchUpdateOrganizationsOfInterestMetadata)` 
What do you think about it?

Reply:
Updated wihtin https://github.com/teamdigitale/io-app/pull/1409/commits/97303adcf1dbbde16b4c8a6be32732937f18f978.
I think the condition will be always true: the "save" button is enabled only when the user selection is changed.